sopel/config/core_section.py

```diff
@@ -373,6 +373,25 @@ class CoreSection(StaticSection):
 
     """
 
+    db_url = ValidatedAttribute('db_url')
+    """A raw database URL.
+
+    If this option is present, Sopel will ignore **all** other ``db_*``
+    settings and use this option's value only.
+
+    Any supported database type *other than SQLite* can be used this way.
```

Comment:
I'm pretty sure SQLite can be used this way too: `sqlite:///relative/path/to/database.sqlite` should work.

The doc for [`make_url`](https://docs.sqlalchemy.org/en/14/core/engines.html#sqlite) seems to agree with me.

Reply:
This was definitely left over from a previous revision. Current one is indeed supposed to support SQLite too.

Whoops!